_posts/2018-12-19-map-puts-shorthand.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Elixir: Map Puts Shortand"
+date:   2018-12-17 07:37:29 +0000
+categories: elixir
+author: Valerie Dryden
+---
+
+In Elixir, we can use the `Map.puts` function to take an existing map and return a new map with either an updated value for a key, or to add a new key / value pair.
+
+cats = %{barnaby: 'red', boba: 'white', max: 'black'}
```

Comment:
Question: do we have code blocks enabled in this blog yet? If so tick, tick, tick!

Reply:
Oops! Good catch :) Have updated.